guides/Libraries.md

```diff
@@ -113,5 +113,13 @@ General Lua libraries:
       <td><a href="https://github.com/kikito/tween.lua">tween.lua</a></td>
       <td>A tweening library for Lua.</td>
     </tr>
+    <tr>
+      <td><a href="https://github.com/leafo/moonscript">Moonscript</a></td>
+      <td>Alternative scripting language. Trans-compiles into Lua.</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/pigpigyyy/Yuescript">Yuescript</a></td>
+      <td>Moonscript dialect. Trans-compiles into Lua.</td>
+    </tr>
```

Comment:
These feel kind of different than the other libraries in the table -- they provide a different development language instead of exposing a set of features.  I think it would make more sense to keep these links on the new language guide.

Reply:
Agreed.
Especially since there are dozens of transcompilers for lua.

Reply:
I have removed them again.
Maybe you can cherry pick files when merging to avoid to add and delete again noise.